go-cosmwasm/api/lib_test.go

```diff
@@ -96,6 +96,7 @@ func binaryAddr(human string) []byte {
 }
 
 func TestInstantiate(t *testing.T) {
+	t.SkipNow()
 	cache, cleanup := withCache(t)
 	defer cleanup()
 
```

Comment:
why are all these tests skipped?

Reply:
These are cosmwasm tests that we never ran and want to start running, but they're calling wasm without encryption. I'm not sure whether we need to fix them to leave it like that.
There are other tests in this module that do pass.

Reply:
aha, that makes sense :+1: